charms/istio-pilot/charmcraft.yaml

```diff
@@ -8,8 +8,8 @@ bases:
       channel: "20.04"
 parts:
   charm:
-    charm-python-packages: [setuptools, pip]
-    build-packages: [git]
+    charm-python-packages: [setuptools, pip, jsonschema, cryptography, cffi]
+    build-packages: [git, rustc, cargo, libffi-dev, libssl-dev, pkg-config]
```

Comment:
The cryptography install returned a message asking for this to be installed in the os, otherwise it failed.
On the tls certs library they don't mention it, but they also base the dependencies on py3.10, which can have less dependencies (or more, depending).